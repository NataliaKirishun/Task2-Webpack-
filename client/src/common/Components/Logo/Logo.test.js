import React from 'react';
import { shallow } from 'enzyme';
import Logo from './Logo';

describe('Logo Snapshot', () => {
    it('Logo Component should render correctly', () => {
        const component = shallow(<Logo />);
        expect(component).toMatchSnapshot();
    });
});