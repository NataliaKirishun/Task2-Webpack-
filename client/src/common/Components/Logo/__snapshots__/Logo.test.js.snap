// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Snapshot Logo Component should render correctly 1`] = `
<styled.a>
  netflixroulette
</styled.a>
`;
